openpdf/src/main/java/com/lowagie/text/Image.java

```diff
@@ -697,8 +697,15 @@ public static Image getInstance(java.awt.Image image, java.awt.Color color,
                                 transparency[0] = transparency[1] = (transparentPixel >> 16) & 0xff;
                                 transparency[2] = transparency[3] = (transparentPixel >> 8) & 0xff;
                                 transparency[4] = transparency[5] = transparentPixel & 0xff;
+                                for (int prevPixel = 0; prevPixel < j; prevPixel++) {
```

Comment:
Can you please change the name of this variable to something more descriptive? (just find another name, please)